zerver/webhooks/patreon/doc.md

```diff
@@ -0,0 +1,14 @@
+Get Zulip notifications for your Patreon events!
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to the url in Patreon to create [incoming webhooks](https://www.patreon.com/portal/registration/register-webhooks).
+
+1. Enter the URL constructed above in the space given.
+   Click the **+** button.
+
+{!congrats.md!}
+
+![](/static/images/integrations/patreon/001.png)
```

Comment:
File names are case sensitive; you have committed `001.PNG` rather than `001.png`.